### PR TITLE
chore: handle non tensor outputs

### PR DESCRIPTION
**Pull request by peri044**
# Description

Fix a bug when the module outputs are non-tensors (example: torch.ops.aten.sym_size will return integers)

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project (You can use the linters)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

File: py/torch_tensorrt/dynamo/conversion/_conversion.py [hunk 1]
```diff
@@ -39,6 +38,8 @@ def infer_module_output_dtypes(
     # such as aten.sum - such outputs can be truncated
     output_dtypes = []
     for output in module_outputs:
+        if not isinstance(output, torch.Tensor):
+            output = torch.tensor(output)
         if truncate_long_and_double and output.dtype == dtype.float64:
```
Comment (narendasan) [inline py/torch_tensorrt/dynamo/conversion/_conversion.py:43]: make `elif`?

File: py/torch_tensorrt/dynamo/conversion/_conversion.py [hunk 2]
```diff
@@ -39,6 +38,8 @@ def infer_module_output_dtypes(
     # such as aten.sum - such outputs can be truncated
     output_dtypes = []
     for output in module_outputs:
+        if not isinstance(output, torch.Tensor):
+            output = torch.tensor(output)
```
Comment (narendasan) [inline py/torch_tensorrt/dynamo/conversion/_conversion.py:42]:
```py
output_ = output
if not instance(output, torch.Tensor): 
    output_ = output
```

```

Comment (narendasan) [reply py/torch_tensorrt/dynamo/conversion/_conversion.py:42]: Consider filtering types, like isinstance(Scalar) or something, throw error otherwise
